src/category_theory/limits/shapes/images.lean

```diff
@@ -541,4 +541,24 @@ instance has_image_maps_of_has_strong_epi_images [has_strong_epi_images C] :
 
 end has_strong_epi_images
 
+variables [has_strong_epi_mono_factorisations.{v} C]
+variables {X Y : C} (f : X ⟶ Y)
+
+/--
+If `C` has strong epi mono factorisations, then the image is unique up to isomorphism, in that if
+`f` factors as a strong epi followed by a mono, this factorisation is essentially the image
+factorisation.
+-/
+def image.iso_strong_epi_mono (I' : C) (e : X ⟶ I') (m : I' ⟶ Y) (comm : e ≫ m = f) [strong_epi e] [mono m] :
```

Comment:
Perhaps some of the explicit arguments could be made implicit?

Reply:
Sure - I made `f` and `I'` implicit, I found it very useful to have `e` and `m` explicit though so I'll leave those as-is.